_pages/how-we-work/changing-your-name.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Changing your name in Google Apps
+---
+
+TTS does our best to make everyone feel welcome. During the onboarding process, we can't always ensure that your preferred name is reflected in Google Apps. You can update your records so that people who look at your name on emails and in Google Meet are reminded to use the name that makes you most comfortable.
+
+## Changing your name
+
+You can change your name by following these steps:
+ - Connect to the [GSA network or a VPN]({{ site.baseurl }}/networks/)
+ - ***Ensure your PIV card reader is inserted into your computer.*** When logging in to GCIMS, you will be asked to choose a security certificate to verify your identity. If you choose the wrong one, you will lose access to GCIMS and have a difficult time regaining access.  Choose carefully!
+ - Go to [GCIMS](https://gcims.gsa.gov)
+ - You will be asked to choose a digital certificate. If your PIV card is inserted, there will be two certificates listed with your name on them. Choose the one that has "Usage" that is ***only*** "Digital Signature"
```

Comment:
This was a little confusing when VPNed into Windows, because neither option had that specific language. It turned out that I needed to just select the first option.

Reply:
Fair! Updated. Tried to balance "the first option probably works" with "here's how you can be sure".  Also changed the page title so it can be more easily expanded to changing names in other systems if/when those situations crop up.